src/components/CurrentListing.tsx

```diff
@@ -5,27 +5,35 @@ import React from "react"
 
 const Listing = () => {
   return (
-    <div className="flex min-h-screen w-full  rounded-lg border-amber-600 border-2	border-solid	">
-      <div className="w-6/12 rounded-lg border-white border-2	border-solid">
-        <p>Some Image</p>
-      </div>
-      <div className="w-6/12 grid  justify-center rounded-lg border-amber-600 border-2	border-solid">
-        <p className="self-end w-fit	">
+    <div className="flex min-h-52	w-full">
+      <div
+        className="w-6/12 p-52 flex rounded-lg bg-cover bg-center m-5"
+        style={{
+          backgroundImage: `url("https://images.unsplash.com/photo-1576941089067-2de3c901e126?w=800&auto=format&fit=crop&q=60&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTZ8fGhvdXNlfGVufDB8fDB8fHww")`,
+        }}
+      ></div>
+
+      <div className="grid w-6/12 grid p-32 justify-items-center ">
+        <p className="self-center w-fit mb-3">
           mi proin sed libero enim sed faucibus. Orci dapibus ultrices in
           iaculis nunc. Quisque id diam vel quam elementum pulvinar etiam non
           quam. Ornare aenean euis mi proin sed libero enim sed faucibus. Orci
           dapibus ultrices in iaculis nunc. Quisque id diam vel quam elementum
           pulvinar etiam non quam. Ornare aenean euis
         </p>
-        <button>Inquire Button</button>
+
+        <button className="py-2 px-5 h-fit w-fit rounded-lg bg-blue-500 border-blue-500 border-2 border-solid">
+          Inquire
```

Comment:
This button become its own component soon so instead of styling it here in this file we can just call `<Button><Button/>` with all this styling.